packages/react-devtools-shared/src/hooks/__tests__/updateMockSourceMaps.js

```diff
@@ -323,7 +323,11 @@ async function bundle() {
     plugins: [
       rollupResolve(),
       commonjs(),
-      babel({presets: ['@babel/preset-react'], sourceMap: true}),
+      babel({
+        presets: ['@babel/preset-react'],
+        sourceMap: true,
+        babelHelpers: 'bundled',
```

Comment:
configure this option explicitly with its default value. https://github.com/rollup/plugins/tree/master/packages/babel#babelhelpers